### PR TITLE
data: Younus-Saberi

### PR DESCRIPTION
**Pull request by Younus-Saberi**
Creating my first LinkFree page
- added tags
- added socials icon
- added link to my social

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5950"></a>

### CODE REVIEW

**Review by Pradumnasaraf — DISMISSED** [review]

File: data/Younus-Saberi.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "url": "https://www.youtube.com/@younussaberi" 
+     }
+  ],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/younussaberi",
+      "icon": "FaTwitter"
+    },
+  ],
```
Comment (Pradumnasaraf) [inline data/Younus-Saberi.json:33]:
```suggestion
    }
  ]
```
```suggestion
    }
  ]
```

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

Remove the `,` comma. Otherwise Looks good

**Comment by Younus-Saberi** [comment]
> Hey,
> 
> Remove the `,` comma. Otherwise Looks good

Thanks, Pradumnasaraf
Before making a PR, I was validating the json on the playground but was getting an error message 
I was not sure where I was making a mistake.

Screenshot 2023-04-09 220634

Was it coming due to my invalid json format?

**Review by khushalarora — DISMISSED** [review]
Looks good, thanks :tada:

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**